### PR TITLE
highest lowest total score

### PR DESCRIPTION
**Pull request by AuroraTuring**
Adds methods to allow stat_tracker to search through an array of summed game scores, to find the highest and lowest total scores.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
   def self.from_csv(locations)
     games = Game.create_from_csv(locations[:games])
     teams = Team.create_from_csv(locations[:teams])
     game_teams = GameTeam.create_from_csv(locations[:game_teams])
     StatTracker.new(games, teams, game_teams)
   end
 
-end
\ No newline at end of file
+  def highest_total_score
```
Comment (wallacebilly1) [inline lib/stat_tracker.rb]: I think we'll need to change this from an instance method run on a StatTracker object to one that lives within the class itself, but this is a great start to it!

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -10,4 +10,4 @@
 
 RSpec.configure do |config|
   config.formatter = :documentation
-end
```
Comment (wallacebilly1) [inline spec/spec_helper.rb:13]: We'll definitely want to make sure we're writing tests first for our future methods.  Makes it harder to test a PR to make sure it's good to merge.

**Review by wallacebilly1 — COMMENTED** [review]
I think we should hold off merging this until we get tests written for the methods and also meet to discuss where our methods should live.

File: lib/stat_tracker.rb (hunk 1 above)
Comment (AuroraTuring) [reply lib/stat_tracker.rb]: According to our stat_tracker_spec it seems the tests should be written in stat_tracker, but I may be misunderstanding.

File: spec/spec_helper.rb (hunk 2 above)
Comment (AuroraTuring) [reply spec/spec_helper.rb:13]: Definitely got my tests written now, sorry for misunderstanding the assignment! I caught the bit about writing tests after my initial pull request.

**Comment by AuroraTuring** [comment]
I resolved my merge conflicts as well, I guess double check you approve of the changes! I included the dummy file replacement fix in that too.

**Comment by AuroraTuring** [comment]
Will push again later when my tests are passing.

**Pull request closed without merging**